### ISSUE TITLE
IPCC says we also need carbon removal. Are those covered by existing carbon markets?

### ISSUE THREAD

**Issue by aboodman** [aboodman/climate#4]

**Comment by snarfed** [comment]
by carbon markets, do you mean, jurisdictions that have imposed prices on carbon? there are a few different mechanisms - tax, cap and trade, etc - but they rarely mandate specific mechanisms. i don't know specifically which ones allow negative emissions, but hopefully they all do.

(i don't know of much negative emissions that's actually deployed in industry, at any scale, as opposed to just research. do you? direct air capture, biochar, olavine weathering, etc are all mainly still science projects, right? _scrubbers_ are absolutely used in industry right now, but that's arguably more emissions reduction than negative emissions per se.)

the biggest problem here is probably that prices on carbon at all, any kind, are still the minority across the world. i think the single biggest positive impact the US could have on climate change is probably to introduce a national price on carbon, regardless of what that price is. even $.01/ton would be a huge deal, in the 0 to 1 sense. sadly i'm not holding my breath right now at least.

**Comment by aboodman** [comment]
> by carbon markets, do you mean, jurisdictions that have imposed prices on carbon?

Yes, that's what I mean. In places where there is a market for carbon credits, can one create those credits w/ negative emissions?

> i don't know of much negative emissions that's actually deployed in industry, at any scale, as opposed to just research. do you?

Well what about something like reforestation. That's presumably net-negative, but there's no (economic) reason to do it without incentives.

**Comment by snarfed** [comment]
> Well what about something like reforestation. That's presumably net-negative, but there's no (economic) reason to do it without incentives.

ah interesting point. the reforestation example is obviously different in that it's very long horizon, very crude impact estimates with big error bars, and high uncertainty due to future possible deforestation or other events, etc. as opposed to something like direct air capture which is the opposite in all ways.

historically i think reforestation has been more part of carbon _offsets_ than carbon markets/negative emissions. we may have more history and experience with offsets, but they aren't as sexy these days. (and i suspect offsets generally don't/can't participate in true carbon markets, hence your q.)
